python_modules/dagster/dagster_tests/definitions_tests/test_sensor_invocation.py

```diff
@@ -1034,6 +1034,31 @@ def after_cursor_partitions_asset_sensor(context):
         after_cursor_partitions_asset_sensor(ctx)
 
 
+def test_multi_asset_sensor_can_start_from_asset_sensor_cursor():
+    @asset
+    def my_asset():
+        return Output(99)
+
+    @multi_asset_sensor(monitored_assets=[my_asset.key])
+    def my_multi_asset_sensor(context):
+        ctx.advance_all_cursors()
+
+    with instance_for_test() as instance:
+        cursor_from_previously_an_asset_sensor = "1234"
```

Comment:
This test is looking great to me so far.

Could we fetch an asset sensor cursor instead of a dummy value? So that if we change the asset sensor cursor logic, this test will catch that the behavior has regressed

You can do something like this to fetch a cursor:
```python
with build_sensor_context(
    instance=instance,
) as ctx:
    result = my_asset_sensor.evaluate_tick(ctx)
    cursor = ctx.cursor
```

Reply:
Absolutely. I didn't use the `with .. as ctx` syntax to avoid nesting my test code as I need data from `ctx` later, but I'm open to change.